app/assets/javascripts/admin_legacy.js

```diff
@@ -0,0 +1,14 @@
+//= require govuk_publishing_components/lib
+//= require jquery-ui
+//= require extensions_legacy/jquery.distinct
+//= require vendor_legacy/jquery/chosen.jquery.js
```

Comment:
It seems unconventional to change the name of a vendor directory. Can we leave it? I imagine it's also not true that these are legacy for the frontend rendering part of the site.

Reply:
yes I did move it back, looks like I missed this one

Reply:
I did move it back in this [commit](https://github.com/alphagov/whitehall/pull/6909/commits/39036b97c8cb22fdbdf13ac8fa8c21119c0e49d1)